.github/workflows/rust-audit.yml

```
@@ -0,0 +1,22 @@
+name: Rust audit
+
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

I think we should consider using the trigger definition suggested by the [audit-check](https://github.com/marketplace/actions/webhook-trigger) store page.

It seems to me that the scheduled run covers when any current dependency changes, and then the `push: paths:` handles when the set of tokio's dependencies changes.

```suggestion
  push:
    paths:
      - '**/Cargo.toml'
      - 'Cargo.lock'
```

---

Thanks! I bundled this suggested change into a squashed commit